win32/tests/threadpool_win32_int/threadpool_win32_int.c

```diff
@@ -998,6 +998,139 @@ static DWORD WINAPI chaos_thread_with_timers_func(LPVOID lpThreadParameter)
     return 0;
 }
 
+//static DWORD WINAPI chaos_thread_with_timers_no_lock_func(LPVOID lpThreadParameter)
+//{
+//    CHAOS_TEST_DATA* chaos_test_data = (CHAOS_TEST_DATA*)lpThreadParameter;
+//
+//    while (InterlockedAdd(&chaos_test_data->chaos_test_done, 0) == 0)
+//    {
+//        int which_action = rand() * 8 / (RAND_MAX + 1);
```

Comment:
>which_action [](http://example.com/codeflow?start=14&length=12)

It is better to have an enum defined with the possible actions.
Then it becomes more readable (instead of having 0, 1, 2, we'd have CHAOS_TEST_TIMER_ACTION_THREADPOOL_OPEN, ... for example

Reply:
done